terraform/modules/data_share_service/vpc.tf

```diff
@@ -1,3 +1,167 @@
+locals {
+  private_subnet_cidr = cidrsubnet(var.vpc_cidr, 1, 0)
+  public_subnet_cidr  = cidrsubnet(var.vpc_cidr, 1, 1)
+
+  private_acl_ingress_rules = [
+    # Allow traffic from loadbalancer
+    {
+      rule_no    = 1
+      from_port  = 8080
+      to_port    = 8080
+      cidr_block = local.public_subnet_cidr
+      action     = "ALLOW"
+      protocol   = "tcp"
+    },
+    # Allow traffic to database from ECS service across private subnets
+    {
+      rule_no    = 2
+      from_port  = 45678
+      to_port    = 45678
+      cidr_block = local.private_subnet_cidr
+      action     = "ALLOW"
+      protocol   = "tcp"
+    },
+    # Deny port 3389 (RDP)
+    {
+      rule_no    = 3
+      from_port  = 3389
+      to_port    = 3389
+      cidr_block = "0.0.0.0/0"
+      action     = "DENY"
+      protocol   = "tcp"
+    },
+    # Allow response to outbound calls
+    {
+      rule_no    = 4
+      from_port  = 1024
+      to_port    = 65535
+      cidr_block = "0.0.0.0/0"
+      action     = "ALLOW"
+      protocol   = "tcp"
+    },
+    {
+      rule_no    = 100
+      from_port  = 0
+      to_port    = 0
+      cidr_block = "0.0.0.0/0"
+      action     = "DENY"
+      protocol   = -1
+    }
+  ]
+  private_acl_egress_rules = [
+    # Allow traffic to database from ECS service
+    {
+      rule_no    = 1
+      from_port  = 45678
```

Comment:
This presumably has to match a few places (e.g. `rds_bastion_host_sg`, `rds_postgres_cluster`) is it worth pulling it out as a variable rather than just hard coding the same number in a few places?